public/app/config/routes.js

```diff
@@ -0,0 +1,58 @@
+(function () {
+    'use strict';
+
+    define(
+        [
+            'page',
+            'querystring',
+            'util/bind'
+        ],
+        function (page, qs, bind) {
```

Comment:
This module (`routes`) is the RWAHS-specific routing / URL configuration.  The `bind` module is the generic part that calls `ko.applyBindings` and runs the various hooks.  These two together used to be the `router` module.